dnf-behave-tests/dnf/plugins-core/config-manager-addrepo.feature

```diff
@@ -0,0 +1,437 @@
+   When I execute dnf with args "config-manager addrepo --id=test --set=baseurl=http://something.com/os/"
+   Then the exit code is 0
+    And file "/etc/yum.repos.d/test.repo" contents is
+        """
+        [test]
+        name=created by dnf5 config-manager
+        enabled=1
+        baseurl=http://something.com/os/
+        """
+
+
+Scenario: test "addrepo" from "baseurl" with user defined destination file name (--save-filename=)
+   When I execute dnf with args "config-manager addrepo --save-filename=test.repo --set=baseurl=http://something.com/os/"
+   Then the exit code is 0
+    And file "/etc/yum.repos.d/test.repo" contents is
+        """
+        [something.com_os_]
+        name=created by dnf5 config-manager
+        enabled=1
+        baseurl=http://something.com/os/
+        """
+
+
+Scenario: "addrepo" from "baseurl" with user defined destination file name, test adding the .repo extension to the filename
+   When I execute dnf with args "config-manager addrepo --save-filename=test --set=baseurl=http://something.com/os/"
+   Then the exit code is 0
+    And file "/etc/yum.repos.d/test.repo" contents is
+        """
+        [something.com_os_]
+        name=created by dnf5 config-manager
+        enabled=1
+        baseurl=http://something.com/os/
+        """
+
+
+Scenario: when run addrepo without URL
+   When I execute dnf with args "config-manager addrepo --id=test_repo --set=excludepkgs=iftop"
+   Then the exit code is 2
+    And stderr contains "One of --from-repofile=<URL>, --set=baseurl=<URL>, --set=mirrorlist=<URL>, --set=metalink=<URL> must be set to a non-empty URL"
+
+
+Scenario: test "addrepo" from "baseurl", set more options
+   When I execute dnf with args "config-manager addrepo --set=baseurl=http://something.com/os/ --set=gpgcheck=1 --set=metadata_expire=7d"
+   Then the exit code is 0
+    And file "/etc/yum.repos.d/something.com_os_.repo" contents is
+        """
+        [something.com_os_]
+        name=created by dnf5 config-manager
+        enabled=1
+        baseurl=http://something.com/os/
+        gpgcheck=1
+        metadata_expire=7d
+        """
+
+
+Scenario: test "addrepo" from "baseurl", set option multiple times with the same value is OK
+   When I execute dnf with args "config-manager addrepo --set=baseurl=http://something.com/os/ --set=gpgcheck=1 --set=metadata_expire=7d --set=gpgcheck=1"
+   Then the exit code is 0
+    And file "/etc/yum.repos.d/something.com_os_.repo" contents is
+        """
+        [something.com_os_]
+        name=created by dnf5 config-manager
+        enabled=1
+        baseurl=http://something.com/os/
+        gpgcheck=1
+        metadata_expire=7d
+        """
+
+
+Scenario: test "addrepo" from "baseurl", set option more times with different value
+   When I execute dnf with args "config-manager addrepo --set=baseurl=http://something.com/os/ --set=gpgcheck=1 --set=metadata_expire=7d --set=gpgcheck=0"
+   Then the exit code is 1
```

Comment:
I am just curious why does this problem ends with RC 1 but previous test `Scenario: when run addrepo without URL` with RC 2?

Reply:
I consider the note as resolved